### ISSUE TITLE
Broken In Build 6429688

### ISSUE THREAD

**Issue by AshleyHolton** [LelouBil/PhasBypass#5]
Bypass seems to be detected after the unity splash screen.

(side question: how are you removing the unicode function names and numbering them?

**Comment by XenoSnowFox** [comment]
yeah I believe it's broken from the latest game update. 

I'm not 100% sure of how this all works, but I have a feeling there's been a new form of obfuscation being applied to the Phasma-code. (i could be wrong here). That said I'd been also keen to know how one would actually work out where they need to hook functions in (eg how to work out what the unknown functions might do).

**Comment by AshleyHolton** [comment]
@XenoSnowFox Fixed in my fork. Lots of that unicode rubbish, but I have no idea how to prune that effectively.

**Comment by LelouBil** [comment]
The names of the functions were always obfuscated like that, but il2cppinspector replaced them with underscores

**Comment by AshleyHolton** [comment]
Odd. il2cppinspector is what generated the unicode. Is there a command line parameter I should be using?

**Comment by XenoSnowFox** [comment]
yeah, I got the unicode symbols aswell.

**Comment by XenoSnowFox** [comment]
@AshleyHolton , you're update seems to work. Although I had to take out some of the log writes in the string compare function because it was doing a lot of them

**Comment by AshleyHolton** [comment]
@XenoSnowFox Yep. I have removed it from my version too. Just for debugging purposes so not needed.

**Comment by LelouBil** [comment]
> Odd. il2cppinspector is what generated the unicode. Is there a command line parameter I should be using?

The new update of il2cppinspector seems to keep the real unicode characters that's all

**Comment by XenoSnowFox** [comment]
> > Odd. il2cppinspector is what generated the unicode. Is there a command line parameter I should be using?
> 
> The new update of il2cppinspector seems to keep the real unicode characters that's all

So which version would you recommend I use?

**Comment by XenoSnowFox** [comment]
@AshleyHolton how did you go about cleaning up the Unicode characters. I'm not 100% sure of what i'm doing wring :S
